ziti/cmd/create/create_config_test.go

```diff
@@ -33,43 +33,44 @@ const (
 
 func getZitiEnvironmentVariables() []string {
 	return []string{
-		"ZITI_HOME",
```

Comment:
This kind of change is hard to verify visually. Personally, I would do re-ordering like this in a different PR.